src/modules/home/components/category/category-item.tsx

```diff
@@ -0,0 +1,53 @@
+import Typography from '@mui/material/Typography';
+import { type CategoryWithIcon } from '../../types/category-with-icon.type';
+import Box from '@mui/material/Box';
+
+const categoryItemContainerStyles = {
```

Comment:
Let's move the styles to the separate file. You can create another file in the same folder with the name: _filename_.styles.ts. Inside it create an object like so:


```ts
import { MuiStylesObject } from '@src/types/utility.types';

export const categoryStyles = {
     categoryItemContainer: {
       // write you styles here
     }

} satisfies MuiStylesObject
```

of course you will need to create a custom type for that first:

```ts
import { SxProps, Theme } from '@mui/material';

export type MuiStylesObject = Record<string, SxProps<Theme>>;
```